api_docs/drone_doc.py

```diff
@@ -0,0 +1,212 @@
+                            "description": "Speed returned"
+                        }
+                    ],
+                    "returns": "http://auto.schema.org/speed",
+                    "title": "GetSpeed"
+                },
+                {
+                    "@id": ":get_position",
+                    "@type": "Operation",
+                    "expects": "null",
+                    "method": "GET",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Position returned"
+                        }
+                    ],
+                    "returns": "http://schema.org/geo",
+                    "title": "GetPosition"
+                },
+                {
+                    "@id": ":get_battery_status",
+                    "@type": "Operation",
+                    "expects": "null",
+                    "method": "GET",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Battery status returned"
+                        }
+                    ],
+                    "returns": "http://schema.org/fuelCapacity",
+                    "title": "GetBatteryStatus"
+                },
+                {
+                    "@id": ":get_model",
+                    "@type": "Operation",
+                    "expects": "null",
+                    "method": "GET",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Model returned"
+                        }
+                    ],
+                    "returns": "http://schema.org/model",
+                    "title": "GetModel"
+                },
+                {
+                    "@id": ":get_status",
+                    "@type": "Operation",
+                    "expects": "null",
+                    "method": "GET",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Status returned"
+                        }
+                    ],
+                    "returns": "http://schema.org/status",
+                    "title": "GetStatus"
+                },
+                {
+                    "@id": ":get_temperature",
+                    "@type": "Operation",
+                    "expects": "null",
+                    "method": "GET",
+                    "possibleStatus": [
+                        {
+                            "code": 200,
+                            "description": "Temperature returned"
+                        }
+                    ],
+                    "returns": "http://schema.org/Float",
+                    "title": "GetTemperature"
+                },
+                {
+                    "@id": ":recharge",
```

Comment:
Not sure about this one 😅 ,as recharge will be handled internally by the drone who will make POST requests?